### PR TITLE
Feat 1.0.0

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

Update to 1.0.0

## Test Plan

Manual test

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ group :development, :test do
     gem "rubocop"
 end
 
-gem "appwrite", "~> 6.0.0"
+gem "appwrite", "~> 7.0.0"
```
Comment (Meldiron) [inline Gemfile:9]: Just double checking, does this mean it will pull 7.1?

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+FROM ruby:3.1
+
 COPY Gemfile Gemfile
-RUN bundle install --no-deployment
-COPY . .
+
+RUN gem install bundler && bundle install
+
+COPY lib lib
+COPY resources resources
```
Comment (Meldiron) [inline Dockerfile:8]: Is this nessessary in Dockerfile, or should that be part of `volumes` of docker-compose?

**Review by Meldiron — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (abnegate) [reply Gemfile]: Updated now to use >=7.1 < 7.2

File: Dockerfile (hunk 2 above)
Comment (abnegate) [reply Dockerfile:8]: Better to have it here so you can run without compose or any volume mounts if you want. It is mounted when using compose too though

**Review by Meldiron — APPROVED** [review]

**Pull request merged**